src/Catalyst.Core.Modules.Consensus/Deltas/DeltaCache.cs

```diff
@@ -46,18 +51,34 @@ public class DeltaCache : IDeltaCache, IDisposable
         private readonly Func<MemoryCacheEntryOptions> _entryOptions;
         public Cid GenesisHash { get; set; }
 
-        public static string GetLocalDeltaCacheKey(CandidateDeltaBroadcast candidate)
-        {
-            return nameof(DeltaCache) + "-LocalDelta-" + MultiBase.Encode(candidate.Hash.ToByteArray(), "base32");
-        }
+        public static string GetLocalDeltaCacheKey(CandidateDeltaBroadcast candidate) { return nameof(DeltaCache) + "-LocalDelta-" + MultiBase.Encode(candidate.Hash.ToByteArray(), "base32"); }
+
+        public static Address TruffleTestAccount = new Address("0xb77aec9f59f9d6f39793289a09aea871932619ed");
 
         public DeltaCache(IHashProvider hashProvider,
             IMemoryCache memoryCache,
             IDeltaDfsReader dfsReader,
             IDeltaCacheChangeTokenProvider changeTokenProvider,
+            IStorageProvider storageProvider,
+            IStateProvider stateProvider,
+            ISnapshotableDb stateDb,
             ILogger logger)
         {
-            var genesisDelta = new Delta {TimeStamp = Timestamp.FromDateTime(DateTime.MinValue.ToUniversalTime())};
+            stateProvider.CreateAccount(TruffleTestAccount, 1_000_000_000.Kat());
+
+            storageProvider.Commit();
+            stateProvider.Commit(CatalystGenesisSpec.Instance);
+
+            storageProvider.CommitTrees();
+            stateProvider.CommitTree();
+
+            stateDb.Commit();
+
+            var genesisDelta = new Delta
+            {
+                TimeStamp = Timestamp.FromDateTime(DateTime.UnixEpoch),
```

Comment:
Nice spotting as prior DateTime.Min returns 0* 0Seconds on windows while it returns 0* 1 seconds on Linux.